### PR TITLE
styles for unsupported network in wallet

### PR DESCRIPTION
**Pull request by Tulun**
Signed-off-by: Tulun <jaykiraly@gmail.com>
Screen Shot 2021-11-07 at 2 47 29 PM

### CODE REVIEW

File: src/components/Wallet/Wallet.tsx [hunk 1]
```diff
@@ -18,19 +24,29 @@ const Wallet: React.FC = () => {
     { skip: !isConnected }
   );
 
-  if (!isConnected || !account || !chainId) {
+  console.log("values", isConnected, chainId, account);
```
Comment (daywiss) [inline src/components/Wallet/Wallet.tsx:27]: Remove

File: src/components/Wallet/Wallet.tsx [hunk 2]
```diff
@@ -18,19 +24,29 @@ const Wallet: React.FC = () => {
...
+  if (account && !isConnected && !chainId) {
```
Comment (daywiss) [inline src/components/Wallet/Wallet.tsx:28]: This logic changed, is this right?

Comment (Tulun) [reply src/components/Wallet/Wallet.tsx:28]: yeah when the network is unsupported, some of these values are undefined -- but the account *is*.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**